bundles/org.openhab.ui/web/src/pages/settings/rules/rule-edit.vue

```diff
@@ -327,7 +327,8 @@ export default {
             destroyOnClose: true,
             closeTimeout: 2000
           }).open()
-          this.$f7router.navigate(this.$f7route.url.replace('/add', '/' + this.rule.uid), { reloadCurrent: true })
+          this.dirty = false
```

Comment:
Hi! Seems I missed that!
Suggesting to move that new line to line 324, which is where the save actually succeeded. That way, the new and the (misplaced) original can be merged into one.

Reply:
Good point, thanks